chapters.md

```diff
@@ -18,26 +18,20 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 Three `Edition Flags` are defined to describe the behavior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
 
 
-#### `EditionFlagHidden`
+#### EditionFlagHidden
 
 The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`: if `EditionFlagHidden` is set to `true`, its `Child ChapterAtoms Elements` MUST also be interpreted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden Flags`. If `EditionFlagHidden` is toggled by a `Control Track` to `false`, the `ChapterFlagHidden Flags` of the `Child ChapterAtoms Elements` SHALL determine whether the `ChapterAtom` is hidden or not.
 
 
-#### `EditionFlagDefault`
+#### EditionFlagDefault
 
-It is recommended that only one Edition's `EditionFlagDefault Flag` is set to `true`. If more then one `Default Edition` is present, the first `Default Edition` MUST be used.
-If an `EditionFlagDefault Flag` is set to `true`, this `Edition` MUST be used. When all `EditionFlagDefault Flags` are set to `false`, the first Edition MUST be used.
+It is RECOMMENDED that no more than one `Edition` have an `EditionFlagDefault Flag` set to `true`. The first `Edition` with the `EditionFlagDefault Flag` set to `true` is the `Default Edition`. When all `EditionFlagDefault Flags` are set to `false`, then the first `Edition` is the `Default Edition`.
 
```

Comment:
Here, I have made some tests with `EditionFlagDefault` set to `true` AND the `EditionFlagHidden` set to `true`.
Simple mkv with 3 editions. 2. edition is set to default and hidden.
The default Edition is not used! Instead the first edition is played.
If the 3. edition deflaut Flag also set to 'true', then the 3. edition is used.

Maybe we should mention this behaviour.

If the `Default Edititon's` 'EditionFlagHidden' is set to 'true' then the first `Edition` is the `Default Edition`.
When more then one  `Edition` `EditionFlagDefault Flag` is set to `true` then the first `Default Edition` is used.

Reply:
@hubblec4: I updated the language to say:
> The first `Edition` with both the `EditionFlagDefault Flag` set to `true` and the `EditionFlagHidden Flag` not set to `false` is the `Default Edition`.

Does this address your comment?